Connected web3 provider at 10.195.2.37:8001
Date.now(): 1579246188556 startTime: 1579246284299 delay: 95746
Deploying smart contract from 0x232ee988523d3f8d7c18c2ae3c3b73e11e80cd0f
TX HASH:
 0x3d077710be71a5ebde4125f351b83a54f6f5513d8bccce879fc4b4626ca4c651
RECEIPT:
 {
  blockHash: '0x0a44cd1b9dafa69f9fb6db783150699275de5323d301af5be9113f9975687e0a',
  blockNumber: 75,
  contractAddress: '0x476b19Bb83c5DE9C2f09052b1C659304178C1E64',
  cumulativeGasUsed: 218495,
  from: '0x232ee988523d3f8d7c18c2ae3c3b73e11e80cd0f',
  gasUsed: 218495,
  logs: [],
  logsBloom: '0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000',
  status: true,
  to: null,
  transactionHash: '0x3d077710be71a5ebde4125f351b83a54f6f5513d8bccce879fc4b4626ca4c651',
  transactionIndex: 0
}
Contract at: 0x476b19Bb83c5DE9C2f09052b1C659304178C1E64

Evaluating  4000txs...

Sending TXS started at 1579246284413 ...
startingBlock: 170
... Sending TXS finished at 1579246385641
Total send duration: 101228
... Waiting for mining process to finish! ...
(node:16485) UnhandledPromiseRejectionWarning: Error: Failed to check for transaction receipt:
{}
    at Object._fireError (/home/ubuntu/benchprezz/node_modules/web3-core-method/node_modules/web3-utils/src/index.js:56:17)
    at /home/ubuntu/benchprezz/node_modules/web3-core-method/src/index.js:260:23
    at runMicrotasks (<anonymous>)
    at runNextTicks (internal/process/task_queues.js:58:5)
    at processTimers (internal/timers.js:472:9)
(node:16485) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:16485) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
ERROR:
 Error: Failed to check for transaction receipt:
{}
    at Object._fireError (/home/ubuntu/benchprezz/node_modules/web3-core-method/node_modules/web3-utils/src/index.js:56:17)
    at /home/ubuntu/benchprezz/node_modules/web3-core-method/src/index.js:260:23
    at runMicrotasks (<anonymous>)
    at runNextTicks (internal/process/task_queues.js:58:5)
    at processTimers (internal/timers.js:472:9)
========================================================
========================================================

/home/ubuntu/benchprezz/benchmark/tpsUnsigned.js:113
                throw 'Blockchain capabilities EXCEEDED - ABORTING test run!';
                ^
Blockchain capabilities EXCEEDED - ABORTING test run!
